2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,15 @@
 */
 
```

Comment:
This looks good 👍 
One suggestion - it's a good idea to use `let` or `const` when declaring variables. This code will still work without it, but when you start working on larger programs - you could run into problems.